PianoFromAbove/GameState.cpp

```diff
@@ -1261,7 +1250,7 @@ void MainScreen::JumpTo(long long llStartTime, bool bUpdateGUI)
         auto itPrev = itMiddle - 1;
         int iFound = 0;
         int iSimultaneous = m_vEvents[itPrev - m_vEvents.begin()]->GetSimultaneous() + 1;
-        for (std::vector<MIDIChannelEvent*>::reverse_iterator it(itMiddle); iFound < iSimultaneous && it != m_vEvents.rend(); ++it)
+        for (std::vector<MIDIChannelEvent*>::reverse_iterator it(itBegin); iFound < iSimultaneous && it != m_vEvents.rend(); ++it)
```

Comment:
Why? This change breaks seeking by removing every note that's touching the keyboard at that time.